content/guides/development.md

```diff
@@ -59,6 +60,24 @@ serve
 
 After each compilation, you will need to manually refresh your browser to see the changes.
 
+#### Watch Mode with Chrome DevTools Workspaces
```

Comment:
I thought it would be considered a sub topic for watch, but I guess it didn't work.